tutorials/01-mnist.ipynb

```diff
@@ -42,24 +42,64 @@
    "id": "b6b7f61d",
```

Comment:
<p>The output changed. Looks like the query itself is ambiguous. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/eva/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I have reverted the tutorials for now.